src/test/java/tds/testpackageconverter/converter/mappers/LegacyAdministrationTestPackageBlueprintMapperTest.java

```diff
@@ -18,7 +18,7 @@
 public class LegacyAdministrationTestPackageBlueprintMapperTest extends LegacyTestPackageBaseTest {
 
     @Test
-    public void shouldConvertTestPackageBlueprintSuccessfullyForSingleAssessment() {
+    public void shouldConvertCATTestPackageBlueprintSuccessfullyForSingleAssessment() {
```

Comment:
I was just thinking.. The only thing to make this method name more epic is for it to be longer... and voila!